lib/node_modules/@stdlib/array/base/cuany-by-right/benchmark/benckmark.length.js

```diff
@@ -0,0 +1,106 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var isArray = require( '@stdlib/assert/is-array' );
+var filled = require( '@stdlib/array/base/filled' );
+var pkg = require( './../package.json' ).name;
+// var cuanyByRight = require( './../lib' );
+import cuanyByRight from '@stdlib/array/base/cuany-by-right/lib/main';
```

Comment:
This is incorrect. Please review other packages. No where do we use `import` statements.